### ISSUE TITLE
Some of Tailwind's minimum height classes get ignored

### ISSUE THREAD

**Issue by falkoschindler** [zauberzeug/nicegui#3176]
### Discussed in https://github.com/zauberzeug/nicegui/discussions/3169

<div type='discussions-op-text'>

<sup>Originally posted by **v479038280** June  4, 2024</sup>
This works:
```py
ui.card().classes('min-h-[220px]')
```

This doesn't work:
```py
ui.card().classes('min-h-96')
```
</div>

**Comment by falkoschindler** [comment]
Thanks for bringing this up, @v479038280!
This bug very strange. I tried to reproduce it with Quasar and Tailwind, but it works as expected:
```html
<html>
  <head>
    <link href="https://cdn.jsdelivr.net/npm/quasar@2.16.4/dist/quasar.prod.css" rel="stylesheet" type="text/css" />
    <script src="https://cdn.tailwindcss.com"></script>
  </head>
  <body>
    <div id="q-app" style="display: flex; flex-direction: column; align-items: start">
      <div class="bg-blue-100 min-h-[220px]">Hello</div>
      <div class="bg-blue-100 min-h-96">Hello</div>
    </div>
    <script src="https://cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/quasar@2.16.4/dist/quasar.umd.prod.js"></script>
    <script>
      const app = Vue.createApp({ setup: () => ({}) });
      app.use(Quasar);
      app.mount("#q-app");
    </script>
  </body>
</html>
```

Does anyone have an idea what is going on?

**Comment by python-and-fiction** [comment]
Maybe tailwindcss version is not `3.4.0` or newer.
I found the same problem in tailwindcss:
https://github.com/tailwindlabs/tailwindcss/discussions/12358#discussioncomment-9591980
The solution is upgrading tailwindcss to `3.4.3`.
And i think [falkoschindler](https://github.com/falkoschindler) 's code works well, because tailwindcss version in https://cdn.tailwindcss.com is newer than `3.4.0`(`3.4.4` for now).

**Comment by falkoschindler** [comment]
Oh great, @python-and-fiction!
Changing the Tailwind version in my example from https://github.com/zauberzeug/nicegui/issues/3176#issuecomment-2151463506 to 3.2.0 (which is currently used by NiceGUI 1.4.x) breaks indeed the layout.

```html
    <script src="https://cdn.tailwindcss.com/3.2.0"></script>
```

Since we usually update dependencies for major releases only, I'll schedule this fix for the upcoming 2.0.0.
